test/Microsoft.Azure.WebJobs.Extensions.Kafka.UnitTests/KafkaListenerTest.cs

```diff
@@ -259,7 +318,8 @@ public async Task When_Topic_Has_Multiple_Partitions_Should_Execute_And_Commit_I
                 listenerConfig,
                 requiresKey: true,
                 valueDeserializer: null,
-                NullLogger.Instance
+                NullLogger.Instance,
+                new Mock<FunctionDescriptor>().Object
```

Comment:
nit: if you are not going to setup of verify the mock you can create using 
```c#
Mock.Of<FunctionDescriptor>()
```

But, I believe we can use `new FunctionDescriptor()` here.

Reply:
Sounds awesome, I update it to Mock.Of<FunctionDescriptor>().

Reply:
Done as new FunctionDescriptor()